reasoner/controller/DisjunctionController.java

```diff
@@ -45,10 +45,10 @@
 import static com.vaticle.typedb.core.common.iterator.Iterators.iterate;
 import static com.vaticle.typedb.core.reasoner.controller.ConjunctionController.merge;
 
-public abstract class DisjunctionController<
-        PROCESSOR extends DisjunctionController.Processor<PROCESSOR>,
-        CONTROLLER extends DisjunctionController<PROCESSOR, CONTROLLER>
-        > extends AbstractController<ConceptMap, ConceptMap, ConceptMap, Request, PROCESSOR, CONTROLLER> {
+public abstract class DisjunctionController<OUTPUT,
```

Comment:
ahh but now it's here haha okay!
you should match the style where `OUTPUT` is on the following line and indented to match the other args